docs/svelte-testing-library/setup.mdx

```diff
@@ -4,85 +4,90 @@ title: Setup
 sidebar_label: Setup
 ---
 
-We recommend using [Vitest](https://vitest.dev/) but you're free to use the library
-with any testing framework and runner you're comfortable with.
+We recommend using [Vitest](https://vitest.dev/) but you're free to use the
+library with any testing framework and runner you're comfortable with.
 
 ## Vitest
-1.  Install Vitest and jsdom
 
-    ```
-    npm install --save-dev vitest jsdom
-    ```
+1. Install Vitest and jsdom
 
-2.  Add the following to your `package.json`
+We're using `jdom` here as the test environment, but you can use any other
+options e.g `happy-dom`.
 
-    ```json
-    {
-      "scripts": {
-        "test": "vitest run src",
-        "test:watch": "vitest src"
-      }
-    }
-    ```
+```bash npm2yarn
+npm install --save-dev vitest jsdom
+```
 
-3.  You'll need to compile the Svelte components before using them in Vitest, so
-    you need to install
-    [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte) and Vite
-    
+Optionally install `@vitest/ui` open a UI within a browser window to view and
+interact with your tests.
 
-    ```
-    npm install --save-dev @sveltejs/vite-plugin-svelte vite
-    ```
+```bash npm2yarn
+npm install --save-dev @vitest/ui
+```
 
-4.  Add a `vitest.config.ts` configuration file to the root of your project
+1. Add the test scipts to your `package.json` to run the tests with Vitest
 
-    ```js
-    import { defineConfig } from 'vitest/config';
-    import { svelte } from '@sveltejs/vite-plugin-svelte';
+   ```json
+   {
+     "scripts": {
+       "test": "vitest run",
+       "test:ui": "vitest --ui",
+       "test:watch": "vitest"
+     }
+   }
+   ```
 
-    export default defineConfig({
-        plugins: [svelte({ hot: !process.env.VITEST })],
-        test: {
-            include: ['src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-            globals: true,
-            environment: 'jsdom'
-        }
-    });
+2. To compile the Svelte components before using them in Vitest, you need to
```

Comment:
The word "to" seems redundant to me here, but maybe that's just me :)

Reply:
I think it's fine.